src/main/java/org/cdc/gov/sdp/queue/DatabaseQueueComponent.java

```diff
@@ -33,14 +35,37 @@ public DatabaseQueueComponent(CamelContext context, Class<? extends Endpoint> en
 		super(context, endpointClass);
 	}
 
+	public String getCreateCommand(String tableName) {
+		return ("CREATE TABLE IF NOT EXISTS " + tableName + " (id bigserial primary key,"
+				+ "cbr_id varchar(255) NOT NULL, source varchar(255) NOT NULL, source_id varchar(255) NOT NULL,"
+				+ "source_attributes text default NULL, batch boolean default false, batch_index int default 0,"
+				+ "payload text NOT NULL, cbr_recevied_time varchar (255) NOT NULL, cbr_delivered_time varchar (255) default NULL,"
+				+ "sender varchar (255) default NULL, recipient varchar (255) default NULL, errorCode varchar (255) default NULL,"
+				+ "errorMessage varchar (255) default NULL, attempts int  default 0, status varchar (255) default 'queued',"
+				+ "created_at varchar (255) default NULL, updated_at varchar (255) default NULL)");
+	}
+
+	public void createIfNotExists(DataSource ds, String tableName) throws Exception {
+		Connection conn = null;
+		PreparedStatement ps = null;
+		try {
+			conn = ds.getConnection();
+			ps = conn.prepareStatement(getCreateCommand(tableName));
+			ps.executeUpdate();
+		} finally {
+			ps.close();
```

Comment:
You should check for null with these before closing.  If there is an issue getting a connection or statement then these may not have been created correctly